assets/css/main.css

```diff
@@ -356,6 +356,15 @@ h3 {
   text-align: center;
 }
 
+#serverless-patterns {
+  font-weight: 300;
+}
+
+#orchestration {
```

Comment:
This is problematic, because there is already an element with this id (heading of section 3)

Reply:
Ooh, good catch.

Reply:
Correction has been committed.